sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-apollo/src/main/java/org/apache/shardingsphere/orchestration/center/instance/wrapper/ApolloOpenApiWrapper.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.orchestration.center.instance.wrapper;
+
+import com.ctrip.framework.apollo.core.ConfigConsts;
+import com.ctrip.framework.apollo.openapi.client.ApolloOpenApiClient;
+import com.ctrip.framework.apollo.openapi.client.constant.ApolloOpenApiConstants;
+import com.ctrip.framework.apollo.openapi.dto.NamespaceReleaseDTO;
+import com.ctrip.framework.apollo.openapi.dto.OpenItemDTO;
+import com.google.common.primitives.Ints;
+import org.apache.shardingsphere.orchestration.center.configuration.InstanceConfiguration;
+
+import java.util.Objects;
+import java.util.Properties;
+
+/**
+ * Apollo open api client wrapper.
+ *
+ * @author dongzonglei
+ */
+public final class ApolloOpenApiWrapper {
+    
+    private ApolloOpenApiClient client;
+    
+    private String namespace;
+    
+    private String appId;
+    
+    private String env;
+    
+    private String clusterName;
+    
+    private String administrator;
+    
+    public ApolloOpenApiWrapper(final InstanceConfiguration config, final Properties properties) {
```

Comment:
Does this function need a document comment?

Reply:
The same to above.

Reply:
The same to above.